platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/WikiCommentActivityStreamTestIT.java

```diff
@@ -0,0 +1,87 @@
+package org.exoplatform.platform.qa.ui.platform.wiki;
+
+import org.exoplatform.platform.qa.ui.commons.Base;
+import org.exoplatform.platform.qa.ui.selenium.platform.ActivityStream;
+import org.exoplatform.platform.qa.ui.selenium.platform.HomePagePlatform;
+import org.exoplatform.platform.qa.ui.selenium.platform.ManageLogInOut;
+import org.exoplatform.platform.qa.ui.wiki.pageobject.RichTextEditor;
+import org.exoplatform.platform.qa.ui.wiki.pageobject.SourceTextEditor;
+import org.exoplatform.platform.qa.ui.wiki.pageobject.WikiHomePage;
+import org.exoplatform.platform.qa.ui.wiki.pageobject.WikiManagement;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import static com.codeborne.selenide.Condition.exist;
+import static com.codeborne.selenide.Selectors.byText;
+import static com.codeborne.selenide.Selenide.$;
+import static org.exoplatform.platform.qa.ui.core.PLFData.DATA_PASS2;
+import static org.exoplatform.platform.qa.ui.core.PLFData.DATA_USER1;
+import static org.exoplatform.platform.qa.ui.selenium.Utils.getRandomNumber;
+import static org.exoplatform.platform.qa.ui.selenium.locator.ActivityStreamLocator.ELEMENT_ACTIVITY_STREAM_CONTAINER;
+import static org.exoplatform.platform.qa.ui.selenium.logger.Logger.info;
+import static org.exoplatform.platform.qa.ui.selenium.testbase.LocatorTestBase.ELEMENT_SKIP_BUTTON;
+
+/**
+ * Created by esmaoui on 23/03/2018.
+ */
+
+
+
+@Tag("sniff")
+@Tag("wiki")
+public class WikiCommentActivityStreamTestIT extends Base{
```

Comment:
you can add this TC in WikiPublishActivityTestIT.java .

Reply:
Done